modules/nw-route-specific-annotations.adoc

```diff
@@ -68,7 +68,7 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 |===
 [.small]
 --
-1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. This file is stored in the `var/lib/haproxy/router/whitelists` folder.
+1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. This file is stored in the `var/lib/haproxy/router/allowlists` folder.
```

Comment:
Just to confirm, this is the correct path.

Reply:
Thanks, looking into this.